__tests__/future/AvoidRerendering.js

```diff
@@ -38,4 +38,35 @@ describe('avoid rerendering', () => {
       ['firstName'],
     ]);
   });
+
+  test('avoids unnecessary renders of AutoFields', () => {
```

Comment:
Just to ensure `AutoFields` matches the same rendering behavior as other form fields.